v4/export/block_allow_list_test.go

```diff
@@ -4,19 +4,18 @@ package export
 
 import (
 	"strings"
+	"testing"
 
 	tcontext "github.com/pingcap/dumpling/v4/context"
+	"github.com/stretchr/testify/require"
```

Comment:
Could you please help group this import to L12~L13?

Reply:
Updated.